.github/workflows/iroha2-stable.yml

```diff
@@ -1,41 +0,0 @@
-name: I2::Stable::Publish
-
-on:
-  push:
-    branches: [iroha2-stable]
-
-env:
-  CARGO_TERM_COLOR: always
-
-jobs:
-  publish:
-    runs-on: [self-hosted, Linux]
-    container:
-      image: 7272721/i2-ci:latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: Swatinem/rust-cache@v1
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build and push iroha2:stable image
-        uses: docker/build-push-action@v3
-        with:
-          push: true
-          tags: hyperledger/iroha2:stable
-          context: .
-          file: Dockerfile
-
-      - name: Build and push docker image (load-rs:release)
-        run: |
-          sleep 10s
-          echo "wait to finish other workflow"
-      - uses: convictional/trigger-workflow-and-wait@v1.6.2
-        with:
-          owner: soramitsu
-          repo: iroha2-longevity-load-rs
-          github_token: ${{ secrets.G_ACCESS_TOKEN }}
-          workflow_file_name: load-rs-push-from-stable.yaml
-          ref: master
```

Comment:
This refactor depends on https://github.com/soramitsu/iroha2-longevity-load-rs/issues/27